source/release_notes/4.4.15-15.rst

```diff
@@ -0,0 +1,35 @@
+.. _PSMDB-4.4.15-15:
+
+================================================================================
+Percona Server for MongoDB 4.4.15-15 (2022-07-19)
+================================================================================
+
+:Date: July 19, 2022
+:Installation: :ref:`install`
+
+Percona Server for MongoDB 4.4.15-15 is an enhanced, source available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 4.4.15 Community Edition.
+It supports MongoDB 4.4.15 protocols and drivers.
+
+Release Highlights
+==================
+
+* Support of :ref:`multiple KMIP servers <kmip>` adds failover to your data-at-rest encryption setup.
+* Allow users to set KMIP client certificate password through a flag to simplify the migration from MongoDB Enterprise to |PSMDB|. 
+
+Other improvements and bug fixes introduced by MongoDB and included in Percona Server for MongoDB are the following:
+
+* `SERVER-66433 <https://jira.mongodb.org/browse/SERVER-66433>`_ - This is a v6.0 backport fix to v4.4 that disables opportunistic read targeting (except for specified hedged reads) in order to prevent possible performance problems associated with uneven read distribution across the secondaries.
```

Comment:
It seems to be a mixup with SERVER-65131, please check

Reply:
Thank you! Fixed